packages/algorand-js/src/runtime/runtime.ts

```diff
@@ -0,0 +1,215 @@
+/* eslint sonarjs/no-duplicate-string: 0 */
+/* eslint sonarjs/no-small-switch: 0 */
+import { mkTransaction } from "algob";
+import { ExecParams, TransactionType } from "algob/src/types";
+import { getProgram } from "algob/test/helpers/fs";
+import { assignGroupID, SSCParams, SSCStateSchema } from "algosdk";
+
+import { mockSuggestedParams } from "../../build/test/mocks/txn";
+import { TealError } from "../errors/errors";
+import { ERRORS } from "../errors/errors-list";
+import { Interpreter } from "../index";
+import { BIGINT0 } from "../interpreter/opcode-list";
+import { checkIndexBound, compareArray } from "../lib/compare";
+import { convertToString } from "../lib/parsing";
+import { assertValidSchema, getKeyValPair } from "../lib/stateful";
+import type { Context, SDKAccount, StackElem, State, Txn } from "../types";
+
+export class Runtime {
+  interpreter: Interpreter;
+  store: State;
+
+  constructor (interpreter: Interpreter) {
+    this.interpreter = interpreter;
+    this.store = {
+      accounts: new Map<string, SDKAccount>(),
+      globalApps: new Map<number, SSCParams>()
+    };
+  }
+
+  assertAccountDefined (a?: SDKAccount): SDKAccount {
+    if (a === undefined) {
+      throw new TealError(ERRORS.TEAL.ACCOUNT_DOES_NOT_EXIST);
+    }
+    return a;
+  }
+
+  assertAppDefined (appId: number): SSCParams {
+    const globalDelta = this.interpreter.ctx.state.globalApps.get(appId);
```

Comment:
let's rename `globalDelta` to `appSatate`.

Reply:
Renamed to `app` (inline with previous declarations)